libraries/webInterface/gui/index.js

```diff
@@ -436,6 +643,9 @@ realityServer.updateManageObjects = function (thisItem2) {
                     realityServer.switchClass(thisObject.dom.querySelector('.target'), 'green', 'yellow');
                     realityServer.switchClass(thisObject.dom.querySelector('.target'), 'one', 'targetWidthMedium');
 
+                    setTooltipTextForElement(thisObject.dom.querySelector('.target'), 'Add a target file or activate a world object to finish' +
```

Comment:
It feels like this doing the querySelector manually is equivalent to what the optionalParent argument accomplishes by providing a different querySelector target

Reply:
Yeah I didn't expect to need to use it as often as I did, I'll just go back and change them to standard querySelector syntax